File: sisprocessos-service/src/main/java/br/com/softplan/feature/usuario/model/Usuario.java
package br.com.softplan.feature.usuario.model;

import br.com.softplan.core.model.AbstractEntity;
import lombok.Data;

import javax.persistence.*;
import java.time.LocalDate;

@Data
@Entity
public class Usuario extends AbstractEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    private String nome;
    @Column(unique = true)
    private String cpf;
    private String telefone;
    private LocalDate aniversario;
    private String endereco;
    private boolean ativo;
    @Column(unique = true)
    private String login;
    private String senha;
    @Enumerated(value = EnumType.STRING)
    private PerfilUsuario perfil;

}

File: sisprocessos-web/src/app/usuarios/model/usuario.ts
import {Enum} from '../../shared/model/enum';

export class Usuario {
  id: number;
  nome: string;
  cpf: string;
  telefone: string;
  aniversario: Date;
  endereco: string;
  ativo: boolean;
  login: string;
  senha: string;
  perfil: Enum;
}
